automation/TaskManager.py

```diff
@@ -469,25 +470,27 @@ def _issue_command(
             browser.command_queue.put(command)
 
             # received reply from BrowserManager, either success or failure
-            error_text = None
-            tb = None
-            status = None
+
+            status = Status()
+
             try:
-                status = browser.status_queue.get(True, browser.current_timeout)
+                status.raw_tuple(
+                    browser.status_queue.get(True, browser.current_timeout)
```

Comment:
This should just return a `Status` object, so there should be no need to create this from components.